src/include/grpc_transcoding/message_reader.h

```diff
@@ -89,6 +106,8 @@ class MessageReader {
   // Status
   ::google::protobuf::util::Status status_;
 
+  unsigned char delimiter_[kGrpcDelimiterByteSize] = {0};
```

Comment:
No need to reset the buffer here.  Need to reset it right before using.

Reply:
Done.